File: acg12/app/src/main/java/org/acg12/ui/activity/AboutActivity.java
package org.acg12.ui.activity;

import android.app.Activity;
import android.os.Bundle;
import android.view.MenuItem;

import org.acg12.ui.base.PresenterActivityImpl;
import org.acg12.views.AboutView;

public class AboutActivity extends PresenterActivityImpl<AboutView> {

    @Override
    public void created(Bundle savedInstance) {
        super.created(savedInstance);
    }

    @Override
    public boolean onOptionsItemSelected(MenuItem item) {
        switch (item.getItemId()) {
            case android.R.id.home:
                finish();
                return true;
            default:
                return super.onOptionsItemSelected(item);
        }
    }
}

File: acg12/app/src/main/kotlin/org/acg12/ui/activity/AboutActivity.kt
package org.acg12.ui.activity

import android.os.Bundle
import android.view.View
import org.acg12.config.Constant
import org.acg12.ui.base.PresenterActivityImpl
import org.acg12.views.AboutView

class AboutActivity : PresenterActivityImpl<AboutView>(), View.OnClickListener {

    override fun created(savedInstance: Bundle) {
        super.created(savedInstance)
    }

    override fun onClick(v: View) {
        val id = v.id
        if (id == Constant.TOOLBAR_ID) {
            finish()
        }
    }

    override fun onDestroy() {
        super.onDestroy()
    }
}
